CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ Generally it's the
 a few additions:
 
 * Any code merged into the Ninja codebase which will be part of the main
-  executable must compile as C++03. You may use C++11 features in a test or an
+  executable must compile as C++11. You may use C++14 features in a test or an
   unimportant tool if you guard your code with `#if __cplusplus >= 201103L`.
```

Comment:
change the 201103L identifier to match the C++14 one.

Reply:
Good catch. I'll do that.

Reply:
Done.